README.md

```diff
@@ -1,2 +1,13 @@
-# hw01
-Repository for homework 1
+# Andrew Hill
+---
+## About Me
+---
+I grew up in southern Mississippi and went to college at **Louisiana State University**. After college, I ran an outreach program through Baton Rouge Parks and Recreation aimed at increasing engagement with underserved members of the community. 
+
+My current interests include:
+* Urban land use policies
+* Architecture
+* Historic preservation 
+* Community engagement 
+
+My current work is on the [Pullman Neighborhood](https://www.nps.gov/pull/index.htm) on the South Side of Chicago  
```

Comment:
Nice work including a link.

Reply:
Evaluation HW-01 for Andrew Hill
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X    |              |            |
| **Coding strategy**         |     X    |              |            |
| **Presentation**            |           |       X     |            |
| **Achievement, creativity** |           |       X       |            |
| **Ease of access**          |     X      |              |            |

Remarks:

* Great work on the first homework - clean, simple code that produces a clear README.md with a number of different markdown elements.
* You had a few comments on your pull request, but should include a slightly more in depth description of your changes and workflow, either in the README or as a longer comment on your pull request.